server/resources/resource_management_test.go

```diff
@@ -1252,6 +1259,66 @@ var _ = Describe("Resource manager", func() {
 				Expect(ok).To(BeTrue())
 			})
 		})
+
+		Describe("When tenant_id is blacklisted", func() {
+			BeforeEach(func() {
+				schemaID = "blacklisted_tenant_id"
+			})
+
+			Context("A member", func() {
+				BeforeEach(func() {
+					auth = memberAuth
+				})
+
+				It("Should create own resource when tenant_id isn't provided and not fill it in",
+					func() {
+						err := resources.CreateResource(
+							context, testDB, fakeIdentity, currentSchema, map[string]interface{}{
+								"id":        memberResourceID,
+								"domain_id": domainAID,
+							})
+
+						Expect(err).NotTo(HaveOccurred())
+
+						result := context["response"].(map[string]interface{})
+						resource, ok := result[schemaID]
+						Expect(ok).To(BeTrue())
+						Expect(resource).To(HaveKeyWithValue("id", memberResourceID))
+						Expect(resource).To(HaveKeyWithValue("domain_id", domainAID))
+						Expect(resource).NotTo(HaveKeyWithValue("tenant_id", auth.TenantID()))
```

Comment:
`tenant_id` should be *filled* based on token after the policy check.

Reply:
`tenant_id` should be filled and saved in DB, but not returned in the response, because it is blacklisted in the policy for Member. It can be checked by fetching it directly from db, or by adding a policy for e.g. admin and using it to get the resource.

Reply:
Added a new test to show that `tenant_id` is in fact filled in and saved to the DB.

Reply:
Oh, I see. Thanks for the clarification.